packages/server/.env.example

```diff
@@ -1,9 +1,10 @@
-PROCORE_SERVER=https://app.procore.com
-
-REDIRECT_URL=http://localhost:4050/oauth/procore/callback
 CLIENT_ID=
 CLIENT_SECRET=
 
-HOSTNAME=localhost
-PORT=4050
-SESSION_PASSWORD=asdf
+BASE_URL=https://api.procore.com
+OAUTH_URL=https://login.procore.com
+
+REDIRECT_URI=http://localhost:3000/oauth/procore/callback
```

Comment:
In Ryby Sample App ([routes.rb#L17](https://github.com/procore/Procore-Sample-ROR/blob/c5deb80d952f119553cb5695533ece4873584d57/config/routes.rb#L17)) there is `auth/procore/callback` (not `oauth`)
It would be better to have consistent values.